doc/source/index.rst

```diff
@@ -59,6 +59,7 @@ the base Satpy installation.
     composites
     resample
     enhancements
+    modifiers
```

Comment:
Tempted to say this should go under composites. Thoughts?

Reply:
I don't mind either way.  I can move it if you prefer.

Reply:
Logically composites and modifiers are the most similar and are used at the same time in Satpy data generation/loading. However, `modifiers` are also not a "primary" topic that users need to worry about initially. I guess if you're making other modifications I'd prefer if you moved this.

Reply:
Moved to be a subsection of composites.